<div style="font-size: 18px; text-align: justify;">
  <ul style="list-style-type:none; padding: 0; margin: 0;">
    <li>Universidad Rafael Landívar</li>
    <li>Facultad de Ciencias Económicas y Empresariales</li>
    <li>Edin Del Valle</li>
    <li>Econometría II</li>
  </ul>
</div>


<h1 style="text-align:center;">Analisis Del Consumo Presente</h1>


<div style="font-size: 18px; text-align: justify;">
   <p>¿Por qué es importante aumentar las observacion?.  En el análisis de datos, uno de los factores más importantes de tomar en cuenta para obtener resultados robustos y confiables es el tamaño de la muestra u observaciones. En este caso, el ejemplo que se hizo en la clase fue realizado con 4 observaciones, en las cuales los resultados pueden ser engañosos y menos representativos.</p>

   <p>Para que el modelo de regresión sea sólido y ofrezca resultados fiables, es crucial contar con un tamaño de muestra adecuado. En general, se recomienda al menos 32 observaciones, lo que mejora la precisión de las estimaciones y se incrementa el poder estadístico, permitiendo una mejor validación de los supuestos del modelo.</p>
</div>


In [4]:
# Paso 1: Definir los datos
# Creando un data frame con los datos de consumo y ingreso disponible
data <- data.frame(
  t = 1:32,               # Tiempo (períodos)
  C_t_minus_1 = c(100, 110, 115, 125, 120, 130, 135, 145, 150, 155, 160, 170, +
                    175, 180, 190, 195, 200, 210, 215, 220, 230, 235, 240, 250, 255, +
                    260, 270, 275, 280, 290, 295, 300),  # Consumo en el periodo anterior (C_{t-1})
  
  Y_t = c(150, 160, 170, 180, 175, 185, 190, 200, 210, 220, 225, +
            235, 240, 250, 260, 270, 280, 290, 300, 310, 320, 330, 340, 350, 360, +
            370, 380, 390, 400, 410, 420, 430),    # Ingreso disponible (Y_t)
  
  C_t = c(110, 115, 125, 130, 125, 135, 140, 145, 150, 155, 160, +
            170, 175, 180, 190, 195, 200, 210, 215, 220, 230, 235, 240, 250, 255, +
            260, 270, 275, 280, 290, 295, 300)      # Consumo actual (C_t)
)

# Paso 2: Estimar los parámetros (α y β) usando regresión lineal
# Usamos lm() para realizar una regresión lineal: C_t ~ C_{t-1} + Y_t
modelo <- lm(C_t ~ C_t_minus_1 + Y_t, data = data)

# Ver los resultados de la regresión
summary(modelo)


Call:
lm(formula = C_t ~ C_t_minus_1 + Y_t, data = data)

Residuals:
    Min      1Q  Median      3Q     Max 
-3.2664 -0.7930  0.0852  1.0279  3.0919 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)  7.48001    0.85627   8.736 1.29e-09 ***
C_t_minus_1  0.43761    0.08011   5.463 7.02e-06 ***
Y_t          0.37708    0.05691   6.626 2.91e-07 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 1.393 on 29 degrees of freedom
Multiple R-squared:  0.9995,	Adjusted R-squared:  0.9994 
F-statistic: 2.782e+04 on 2 and 29 DF,  p-value: < 2.2e-16


<div style="font-size: 18px; text-align: justify;">

**Intercept:** Una unidad adicional implica un efecto promedio de 7.48 unidades en el intercepto, aunque este no tiene mucho sentido debido a que representa un consumo autónomo o mínimo, que siempre será el mismo cuando las otras variables sean cero.

</div>
</div>


<div style="font-size: 18px; text-align: justify;">

**C_t_minus_1:** Una unidad adicional en el consumo durante el periodo anterior, aumenta el consumo actual en promedio 0.43761 unidades.

**Y_t:** Por cada unidad adicional el ingresos disponible, aumenta el consumo actual en promedio de 0.37708 unidades. 
</div>
</div>


<div style="font-size: 18px; text-align: justify;">
    
**Std. Error y T Value:**
1. **C_t_minus_1**: El error estándar de \(C_t_{-1}\) es 0.08011, lo que indica una buena precisión en la estimación de este coeficiente. El valor t es 5.463, lo que sugiere que es estadísticamente significativo.

2. **Y_t**: El error estándar de \(Y_t\) es 0.05691. lo cual es relativamente pequeño en comparación con el coeficiente de estimación. Esto indica que la estimación de dicho coeficiente es bastante precisa y que el ajuste es confiable. 

</div>




<div style="font-size: 18px; text-align: justify;">
    
- **P-value:** El p-value de cada coeficiente es menor a 0.05, lo que indica que cada uno es estadisticamente significativo. Es decir que las variables explicativas tiene un efecto real sobre el consumo actual y los resultados del modelo no debe al azar.

</div>


<div style="font-size: 18px; text-align: justify;">

El **residual standard error (RSE)** de 1.393 indica que, en promedio, las predicciones se desvían de los valores observados por 1.393 unidades. Un RSE bajo sugiere que el modelo ajusta bien los datos.

</div>


<div style="font-size: 18px; text-align: justify;">
    
El R-squared es **0.9995**, lo que esta cercano a 1, lo que nos da más fiabilidad para confiar en nuestro modelo. En muchos casos, el modelo siempre debe tener algún nivel de error residual, si nos diera un valor de 1, sería muy dificil que un modelo se ajuste perfectamente a la linea de regresión o a los datos reales. 

El **F-statistic 2.782e+04** es un valor muy alto sugiere que las variables independientes explican bien la variabilidad de la variable dependiente. 

La **p-value del F-statistic es < 2.2e-16.** Como es menor que el nivel de significancia (0.05), esto confirma que existe suficiente evidencia para rechazar la hipótesis nula, lo cual indica que el modelo tiene un efecto real.


</div>